packages/eas-cli/src/graphql/queries/UpdateQuery.ts

```diff
@@ -112,4 +135,142 @@ export const UpdateQuery = {
         .toPromise()
     );
   },
+  async viewUpdateGroupAsync({ groupId }: { groupId: string }): Promise<ViewUpdatesByGroupQuery> {
+    return await withErrorHandlingAsync(
+      graphqlClient
+        .query<ViewUpdatesByGroupQuery, ViewUpdatesByGroupQueryVariables>(
+          gql`
+            query ViewUpdatesByGroup($groupId: ID!) {
+              updatesByGroup(group: $groupId) {
+                id
+                group
+                runtimeVersion
+                platform
+                message
+                createdAt
+                manifestFragment
+                actor {
+                  id
+                  ... on User {
+                    username
+                  }
+                  ... on Robot {
+                    firstName
+                  }
+                }
+                branch {
+                  id
+                  name
+                }
+              }
+            }
+          `,
+          {
+            groupId,
+          },
+          { additionalTypenames: ['Update'] }
+        )
+        .toPromise()
+    );
+  },
+  async viewUpdateGroupsOnBranchAsync({
+    limit,
+    offset,
+    appId,
+    branchName,
+    filter,
+  }: ViewUpdateGroupsOnBranchForAppQueryVariables): Promise<ViewUpdateGroupsOnBranchForAppQuery> {
+    return await withErrorHandlingAsync(
+      graphqlClient
+        .query<ViewUpdateGroupsOnBranchForAppQuery, ViewUpdateGroupsOnBranchForAppQueryVariables>(
+          gql`
+            query ViewUpdateGroupsOnBranchForApp(
+              $appId: String!
+              $branchName: String!
+              $limit: Int!
+              $offset: Int!
+              $filter: UpdatesFilter
+            ) {
+              app {
+                byId(appId: $appId) {
+                  id
+                  updateBranchByName(name: $branchName) {
+                    id
+                    updateGroups(limit: $limit, offset: $offset, filter: $filter) {
+                      id
+                      group
+                      message
+                      createdAt
+                      runtimeVersion
+                      platform
+                      manifestFragment
+                      branch {
+                        id
+                        name
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          `,
+          {
+            appId,
+            limit,
+            offset,
+            branchName,
+            filter,
+          },
+          { additionalTypenames: ['Update'] }
+        )
+        .toPromise()
```

Comment:
It looks like in other queries where the target data is nested it's common to dig in here rather than at the callsite. For example, see `EnvironmentSecretsQuery.byAppIdAsync`. If you still want this to throw when the app ID returns nothing that can also be done in this method.

This would make the result type more closely match the what the name of the function indicates is returned.

(ditto with function below)